aws/templates/application/application_computecluster.ftl

```diff
@@ -159,7 +159,7 @@
                             id=formatDependentSecurityGroupIngressId(
                                 resources["securityGroup"].Id
                                 link.Id)
-                            port=link.Port
+                            port=link.DestinationPort
```

Comment:
I'd prefer to see the destinationPort as an attribute on the target and obtained from there. From what I can see, we have the target occurrence at the point where we need the destination port.